src/plugins/dashboard/public/application/dashboard_app_controller.tsx

```diff
@@ -1109,12 +1111,6 @@ export class DashboardAppController {
         $scope.model.filters = filterManager.getFilters();
         $scope.model.query = queryStringManager.getQuery();
         dashboardStateManager.applyFilters($scope.model.query, $scope.model.filters);
-        if (dashboardContainer) {
-          dashboardContainer.updateInput({
```

Comment:
This is not needed and handled by `refreshDashboardContainer`. Looking forward to this file being refactored